### PR TITLE
bpo-31459: Rename IDLE's module browser from Class Browser to Module Browser.

### PR DESCRIPTION
**Pull request by csabella**
https://bugs.python.org/issue31459

### CODE REVIEW

File: Lib/idlelib/config-keys.def [hunk 1]
```diff
@@ -24,7 +24,7 @@ history-previous=<Alt-Key-p> <Meta-Key-p> <Alt-Key-P> <Meta-Key-P>
 interrupt-execution=<Control-Key-c> <Control-Key-C>
 view-restart=<Key-F6>
 restart-shell=<Control-Key-F6>
-open-class-browser=<Alt-Key-c> <Meta-Key-c> <Alt-Key-C> <Meta-Key-C>
+open-module-browser=<Alt-Key-c> <Meta-Key-c> <Alt-Key-C> <Meta-Key-C>
```
Comment (terryjreedy) [inline Lib/idlelib/config-keys.def]: Changing the virtual event name breaks compatibility existing user customizations.  I don't want to do that, certainly not yet.  So revert all in this file.

File: Lib/idlelib/config.py [hunk 2]
```diff
@@ -633,7 +633,7 @@ def GetCoreKeys(self, keySetName=None):
             '<<interrupt-execution>>': ['<Control-c>'],
             '<<view-restart>>': ['<F6>'],
             '<<restart-shell>>': ['<Control-F6>'],
-            '<<open-class-browser>>': ['<Alt-c>'],
+            '<<open-module-browser>>': ['<Alt-c>'],
```
Comment (terryjreedy) [inline Lib/idlelib/config.py]: And here.

File: Lib/idlelib/editor.py [hunk 3]
```diff
@@ -190,7 +190,7 @@ def __init__(self, flist=None, filename=None, key=None, root=None):
                 flist.dict[key] = self
             text.bind("<<open-new-window>>", self.new_callback)
             text.bind("<<close-all-windows>>", self.flist.close_all_callback)
-            text.bind("<<open-class-browser>>", self.open_class_browser)
+            text.bind("<<open-module-browser>>", self.open_module_browser)
```
Comment (terryjreedy) [inline Lib/idlelib/editor.py]: And here (just the <<open...>> part.

File: Lib/idlelib/mainmenu.py [hunk 4]
```diff
@@ -25,7 +25,7 @@
    ('_New File', '<<open-new-window>>'),
    ('_Open...', '<<open-window-from-file>>'),
    ('Open _Module...', '<<open-module>>'),
-   ('Class _Browser', '<<open-class-browser>>'),
+   ('Module _Browser', '<<open-module-browser>>'),
```
Comment (terryjreedy) [inline Lib/idlelib/mainmenu.py]: and revert <<...>> here.

**Review by terryjreedy — CHANGES_REQUESTED** [review]
Revert config-keys.cfg compatibility break.

**Comment by csabella** [comment]
I didn't expect the Spanish Inquisition!

**Review by terryjreedy — APPROVED** [review]

**Comment by terryjreedy** [comment]
To test, I used both the menu and hot key in Shell with file already open or not, and then both menu and hot key in editor.

**Pull request merged**